Firestore/Source/Remote/FSTSerializerBeta.mm

```diff
@@ -316,20 +313,18 @@ - (GCFSValue *)encodedReferenceValueForDatabaseID:(const DatabaseId *)databaseID
             util::WrapNSStringNoCopy(databaseID->project_id()),
```

Comment:
Could you also fix this log message to just use .c_str() directly?

Reply:
done